packages/fxa-settings/src/components/UnitRowSecondaryEmail/index.stories.tsx

```diff
@@ -173,26 +121,10 @@ storiesOf('Components|UnitRowSecondaryEmail', module)
   })
   .add('Multiple secondary emails set, multiple unverified', () => {
     const emails = [
-      {
-        email: 'johndope@example.com',
-        isPrimary: true,
-        verified: true,
-      },
-      {
-        email: 'johndope2@example.com',
-        isPrimary: false,
-        verified: true,
-      },
-      {
-        email: 'johndope3@example.com',
-        isPrimary: false,
-        verified: false,
-      },
-      {
-        email: 'johndope4@example.com',
-        isPrimary: false,
-        verified: false,
-      },
+      mockEmail('johndope@example.com'),
+      mockEmail('johndope2@example.com', false, false),
```

Comment:
It looks like this one should be `mockEmail('johndope2@example.com', false),` based on the code it is replacing

Reply:
Ah thanks for catching, I'll double check the others in Storybook are correct.